panic in `nodes.(*Service).GetNodes`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xccc522]

goroutine 2906 [running]:
code.vegaprotocol.io/data-node/nodes.(*Service).GetNodes(0x0, 0x1533e68, 0xc00c193590, 0x30, 0x7f58f99da108, 0x30, 0xc00c193590, 0xc0003a16a0)
	/go/src/code.vegaprotocol.io/data-node/nodes/service.go:82 +0x22
code.vegaprotocol.io/data-node/api.(*tradingDataService).GetNodes(0xc0000fa1e0, 0x1533e68, 0xc00c193590, 0xc021e38e80, 0x0, 0x0, 0x0)
	/go/src/code.vegaprotocol.io/data-node/api/trading_data.go:331 +0xab
code.vegaprotocol.io/protos/data-node/api/v1._TradingDataService_GetNodes_Handler.func1(0x1533e68, 0xc00c193590, 0x12c4900, 0xc021e38e80, 0x1168a40, 0xc00c19a2c0, 0x1533e68, 0xc00c193590)
	/go/pkg/mod/code.vegaprotocol.io/protos@v0.41.1-0.20210826163644-b2e3ca672bc7/data-node/api/v1/trading_data.pb.go:8782 +0x8b
code.vegaprotocol.io/data-node/api.remoteAddrInterceptor.func1(0x1533e68, 0xc00c193560, 0x12c4900, 0xc021e38e80, 0xc021e38ea0, 0xc00c072300, 0xc024aacba0, 0x49dd86, 0x12b5020, 0xc00c193560)
	/go/src/code.vegaprotocol.io/data-node/api/server.go:210 +0x16c
code.vegaprotocol.io/protos/data-node/api/v1._TradingDataService_GetNodes_Handler(0x137d500, 0xc0000fa1e0, 0x1533e68, 0xc00c193560, 0xc00c1984e0, 0xc00017de60, 0x1533e68, 0xc00c193560, 0x0, 0x0)
	/go/pkg/mod/code.vegaprotocol.io/protos@v0.41.1-0.20210826163644-b2e3ca672bc7/data-node/api/v1/trading_data.pb.go:8784 +0x150
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000fcc40, 0x153e038, 0xc000274780, 0xc0000c9440, 0xc0000be810, 0x1cd4c58, 0x0, 0x0, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:1286 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0000fcc40, 0x153e038, 0xc000274780, 0xc0000c9440, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:1609 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0002688e0, 0xc0000fcc40, 0x153e038, 0xc000274780, 0xc0000c9440)
	/go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:934 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/pkg/mod/google.golang.org/grpc@v1.38.0/server.go:932 +0x1fd
```

--- a/cmd/data-node/node/node_pre.go
+++ b/cmd/data-node/node/node_pre.go
@@ -17,6 +17,7 @@ import (
 	"code.vegaprotocol.io/data-node/logging"
 	"code.vegaprotocol.io/data-node/markets"
 	"code.vegaprotocol.io/data-node/netparams"
+	"code.vegaprotocol.io/data-node/nodes"
 	"code.vegaprotocol.io/data-node/notary"
 	"code.vegaprotocol.io/data-node/oracles"
 	"code.vegaprotocol.io/data-node/orders"
@@ -204,6 +205,7 @@ func (l *NodeCommand) preRun(_ []string) (err error) {
 	l.eventService = subscribers.NewService(l.broker)
 	l.epochService = epochs.NewService(l.Log, l.conf.Epochs, l.epochStore)
 	l.delegationService = delegations.NewService(l.Log, l.conf.Delegations, l.delegationStore)
+	l.nodeService = nodes.NewService(l.Log, l.conf.Nodes, l.nodeStore, l.epochStore)
 
 	l.broker, err = broker.New(l.ctx, l.Log, l.conf.Broker)
 	if err != nil {

